<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_gui</class>
 <widget class="QDialog" name="client_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="p1_layout">
      <item>
       <widget class="QLabel" name="p1_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player1">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p2_layout">
      <item>
       <widget class="QLabel" name="p2_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player2">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p3_layout">
      <item>
       <widget class="QLabel" name="p3_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player3">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="p4_layout">
      <item>
       <widget class="QLabel" name="p4_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Player 4:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player4">
        <property name="text">
         <string>Leer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Schliessen</string>
   </property>
  </widget>
  <widget class="QLabel" name="ServerIP_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverIP_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>176</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Verbinden</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_edit_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>- - - - - - - - - - - - - - - - - - - - - - -</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
